src/mono/wasm/runtime/cwraps.ts

```diff
@@ -148,9 +148,9 @@ const fn_signatures: SigLine[] = [
     [true, "mono_jiterp_tlqueue_clear", "void", ["number"]],
     [true, "mono_jiterp_begin_catch", "void", ["number"]],
     [true, "mono_jiterp_end_catch", "void", []],
-
+    [true, "mono_interpgo_load_table", "number", ["number", "number"]],
+    [true, "mono_interpgo_save_table", "number", ["number", "number"]],
     ...diagnostics_cwraps,
-    ...legacy_interop_cwraps
```

Comment:
`legacy_interop_cwraps` need to stay, right ?

Reply:
Yes, good catch. I'm not sure what happened here.